### PR TITLE
fix learning rate/dataset in the scripts

### PR DESCRIPTION
**Pull request by mmsori**
# PR Title
- Finetuning script 의 hyperparameter/dataset 변경 

## Added Features
- N/A

## Code Refactoring
- N/A

## Bug Fix
- N/A

## ETC
- Finetuning script 의 hyperparameter/dataset 변경 
    - learning rate : Full Finetuning 기준 1e-5 로 변경 
    - dataset : bitext/Bitext-customer-support-llm-chatbot-training-dataset 로 변경

### CODE REVIEW

**Review by moi-gwangho — APPROVED** [review]
learning rate가 정말 카오스였군여...ㄷㄷ
